### ISSUE TITLE
Extra Binary Sensor for Smart Detection

### ISSUE THREAD

**Issue by Mulle00** [briis/unifiprotect#342]
### New Feature

Is it possible to get an extra binary sensor with every camera when smart detection (person and or vehicle) is activated.

I have a ton of motion detection triggers, but it would be helpful if I could also receive the smart detection which is more filtered.

I used the:
- G4 Doorbell
- G4 Bullet

Thanks in advance

![Naamloos](https://user-images.githubusercontent.com/25512372/142923983-edf1ff13-cb28-4a06-b17b-8109ae0fe41b.png)

**Comment by briis** [comment]
We could look at that. But you can already now get that info, if also check the attribute `event_object` when motion occurs. If that is different from `None Identified` then it is a smart event.

**Comment by Mulle00** [comment]
Thank you, I'll see if I can work with this further

**Comment by RiviqMS** [comment]
I would love this as well, but for the purpose that my motion sensor only triggers for motion capture as defined by the 'motion zone' (configured in Unifi Protect). It seems that it now also triggers on smart events. This gives me too many motion events (for example when integrating with Homekit). I only want this motion sensor to trigger when someone enters my property. Also,  I do not want to decrease the Smart detection zone, as I still want to record those smart events to my Cloudkey that happen just outside my doorway.

**Comment by AngellusMortis** [comment]
We talked about this more and we decided to go ahead and implement this. It would be a full split of motion / smart detections though. The existing Motion binary sensor will _not_ do smart detections anymore and there will be a new sensor (not binary sensor) for "detected object".

HA is pretty strict about not duplicating data, so having the `event_object` attribute _and_ a detected object sensor will create duplicated data.
